build.sbt

```diff
@@ -13,10 +13,10 @@ val commonSettings = Seq(
   organization  := "net.ixias",
   scalaVersion  := "2.12.11",
   resolvers ++= Seq(
-    "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/",
+    "Typesafe Releases" at "https://repo.typesafe.com/typesafe/ivy-releases/",
```

Comment:
`https://repo.typesafe.com/typesafe/releases/` is deprecated.
Update to the latest reference.